pkg/machine/e2e/machine_test.go

```diff
@@ -169,3 +183,34 @@ func copySparse(dst io.WriteSeeker, src io.Reader) (retErr error) {
 	_, err := io.Copy(spWriter, src)
 	return err
 }
+
+func setTmpDir(value string) (string, error) {
+	switch {
+	case runtime.GOOS != "darwin":
+		tmpDir = value
+	case len(value) >= 22:
+		return "", errors.New("path length should be less than 22 characters")
```

Comment:
Nit, so the user knows what path it is: 
```suggestion
		return "", errors.New("$TMPDIR path length should be less than 22 characters")
```

Reply:
small change and fixed